### PR TITLE
release notes for v0.1.0

### PR DESCRIPTION
**Pull request by jklynch**

### CODE REVIEW

**Review by mrakitin — APPROVED** [review]
Thanks, looks good to me. I wonder if we should tag it as v0.1.0. Were there enough changes?

**Comment by jklynch** [comment]
I was also thinking v0.1.0 seems reasonable.

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,13 @@
 Release History
 ***************
 
+v0.1.0 (2020-09-04)
+====================
```
Comment (mrakitin) [inline docs/source/release-history.rst:6]: I think it's too many equal signs here.
```suggestion
===================
```

**Review by mrakitin — COMMENTED** [review]
Looks good, just need to make sphinx happy.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**